docs/contributing.md

```diff
@@ -119,14 +100,16 @@ nox -s watch_docs
 nox -s publish_docs
 ```
 
-## Release New `pipx` Version
+## Releasing New `pipx` Versions
 To create a new release
 
+* update pipx's version in `__main__.py` and regenerate documentation
```

Comment:
Where is the version specified in `__main__.py`?  And why not `setup.py`?

Reply:
Oh I think maybe you mean `main.py` ?

Reply:
And I see the fact that `setup.py` no longer has a hard-coded version.

Reply:
Yup I meant `main.py`, thanks.